NEWS.md

```diff
@@ -1,6 +1,9 @@
 # dv.papo 2.0.6-900
 - Update to provide early error feedback if a sender_id is not available in list of modules.
 
+# dv.papo 2.0.5-900
```

Comment:
The version number does not match the one in Description.

My change would be to go up to 2.0.7-900 in both description and the news md section.

It does not make much sense otherwise.